airo-camera-toolkit/airo_camera_toolkit/cameras/realsense.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations
+
+from typing import Any, Tuple
+
+import numpy as np
+import pyrealsense2 as rs  # type: ignore
+from airo_camera_toolkit.interfaces import RGBCamera
+from airo_camera_toolkit.utils import ImageConverter
+from airo_typing import CameraIntrinsicsMatrixType, NumpyFloatImageType, OpenCVIntImageType
+
+
+class Realsense(RGBCamera):
+    """Initial wrapper around the pyrealsense2 library to use the realsense cameras.
+    Written mainly for use with the D415 and D435.
+
+    Depth images are not yet supported.
+    """
+
+    def __init__(
+        self,
+        resolution: Tuple[int, int] = (1280, 720),
```

Comment:
would prefer to have some variables for the available resolutions, similar to the ZED class https://github.com/airo-ugent/airo-mono/blob/f702d30a7c2651791fc1c49dd54fb97e60a3348e/airo-camera-toolkit/airo_camera_toolkit/cameras/zed2i.py#L55. 

Easier to see what resolutions are available imo

Reply:
I've add the D415 resolutions, I'm not 100% sure these are consistent for all RealSense cameras.